MATLAB/mex_CUDA_win64_MSV2019.xml

```diff
@@ -0,0 +1,278 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<config
+    Name="NVIDIA CUDA Compiler"
+    ShortName="nvcc"
+    Manufacturer="NVIDIA"
+    Version="9.2"
+    Language="CUDA"
+    Priority="A"
+    Location="$NVCC" >
+    <Details
+        CompilerExecutable="$COMPILER"
+        CompilerDefines="$COMPDEFINES"
+        CompilerFlags="$COMPFLAGS"
+        OptimizationFlags="$OPTIMFLAGS"
+        DebugFlags="$DEBUGFLAGS"
+        IncludeFlags="$INCLUDE"
+        LinkerExecutable="$LINKER"
+        LinkerFlags="$LINKFLAGS"
+        LinkerLibraries="$LINKLIBS"
+        LinkerOptimizationFlags="$LINKOPTIMFLAGS"
+        LinkerDebugFlags="$LINKDEBUGFLAGS"
+        CommandLineShell="$VCVARSALLDIR\VCVARSALL.BAT "
+        CommandLineShellArg=" amd64 "
+        CompilerDefineFormatter="--compiler-options=/D%s"
+        LinkerLibrarySwitchFormatter="lib%s.lib;%s.lib"
+        LinkerPathFormatter="/LIBPATH:%s"
+        LibrarySearchPath="$$LIB;$$LIBPATH;$$PATH;$$INCLUDE;$MATLABROOT\extern\lib\$ARCH\microsoft"    />
+    <!-- Switch guide: http://msdn.microsoft.com/en-us/library/fwkeyyhe(v=vs.71).aspx -->
+    <vars
+        CMDLINE100="$COMPILER -c $COMPFLAGS $OPTIM $COMPDEFINES $INCLUDE $SRC -o $OBJ"
+        CMDLINE200="$LINKER $LINKFLAGS $LINKTYPE $LINKOPTIM $LINKEXPORT $LINKLIBS $OBJS /out:$EXE"
+        CMDLINE250="mt -outputresource:$EXE;2 -manifest $MANIFEST"
+        CMDLINE300="del $OBJ $EXP $LIB $MANIFEST $ILK"
+        
+        COMPILER="nvcc"
+        COMPFLAGS="-gencode=arch=compute_30,code=sm_30 -gencode=arch=compute_35,code=sm_35 -gencode=arch=compute_50,code=sm_50 -gencode=arch=compute_52,code=sm_52 -gencode=arch=compute_61,code=sm_61 -gencode=arch=compute_70,code=sm_70 --default-stream per-thread  --ptxas-options=-v --compiler-options=/c,/GR,/W3,/EHs,/nologo,/MD"
+        COMPDEFINES="--compiler-options=/D_CRT_SECURE_NO_DEPRECATE,/D_SCL_SECURE_NO_DEPRECATE,/D_SECURE_SCL=0,$MATLABMEX"
+        MATLABMEX="/DMATLAB_MEX_FILE"
+        OPTIMFLAGS="--compiler-options=/O2,/Oy-,/DNDEBUG"
+        INCLUDE="-I&quot;$MATLABROOT\extern\include&quot; -I&quot;$MATLABROOT\simulink\include&quot;"
+        DEBUGFLAGS="--compiler-options=/Z7"
+        
+        LINKER="link"
+        LINKFLAGS="/nologo /manifest"
+        LINKTYPE="/DLL "
+        LINKEXPORT="/EXPORT:mexFunction"
+        LINKLIBS="/LIBPATH:&quot;$MATLABROOT\extern\lib\$ARCH\microsoft&quot; libmx.lib libmex.lib libmat.lib gpu.lib cudart.lib kernel32.lib user32.lib gdi32.lib winspool.lib comdlg32.lib advapi32.lib shell32.lib  ole32.lib oleaut32.lib uuid.lib odbc32.lib odbccp32.lib"
```

Comment:
Lets remove `gpu.lib` from here. Its a MATLAB library coming from the Parallel Computing Toolbox that is not required (MATLAB sucks at naming, they could have called it `matab_gpu.lib`). Some people don't have this toolbox and they should be able to use TIGRE anyway.

Reply:
I just tested it without gpu.lib, and it is working.
It really sucks at naming libraries.